terraform/bigquery.tf

```diff
@@ -740,3 +740,78 @@ resource "google_bigquery_table_iam_member" "commit_review_status_viewers" {
   role       = "roles/bigquery.dataViewer"
   member     = each.value
 }
+
+# Invocation Comment Status Table / IAM
+
+resource "google_bigquery_table" "invocation_comment_table" {
+  project = data.google_project.default.project_id
+
+  deletion_protection = false
+  table_id            = var.invocation_comment_id
+  dataset_id          = google_bigquery_dataset.default.dataset_id
+  schema = jsonencode([
+    {
+      "name" : "pull_request_id",
+      "type" : "INT64",
+      "mode" : "REQUIRED",
+      "description" : "ID of pull request."
+    },
+    {
+      "name" : "pull_request_html_url",
+      "type" : "STRING",
+      "mode" : "REQUIRED",
+      "description" : "URL of pull request."
+    },
+    {
+      "name" : "processed_at",
+      "type" : "TIMESTAMP",
+      "mode" : "REQUIRED",
+      "description" : "Timestamp of when the event was processed."
+    },
+    {
+      "name" : "status",
+      "type" : "STRING",
```

Comment:
Another field I considered but am less sold on is the message body, might make it easier if we change how things are done later down the line, but I'm like 50/50 whether that is a good idea.